hobby-blog.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Blog Ramblings</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="css" href="stylesheets/styles.css">
+    <!-- for FF, Chrome, Opera -->
+    <link rel="icon" type="image/png" href="./images/chi.png" sizes="16x16">
+    <link rel="icon" type="image/png" href="./images/chi.png" sizes="32x32">
+    <!-- for IE -->
+    <link rel="icon" type="image/x-icon" href="./images/chi.png" >
+    <link rel="shortcut icon" type="image/x-icon" href="./images/chi.png"/>
+  </head>
+  <body>
+    <header>
+    </header>
+    <nav class="top-bar">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="portfolio.html">Portfolio</a></li>
+        <li><a href="hobby-blog.html">Hobby Blarg</a></li>
+      </ul>
+    </nav>
+    <section>
+      <article class="hobby-entries">
+        <p>
+          Angela of <a href="http://www.freckledcanvas.com">Freckled Canvas</a> got my cross stitch that I finished for her over spring break! Thank you, USPS. 
+        </p>
+        <p>
+          <a href="https://www.instagram.com/p/BDtevy2HT-e/">
+          <img border="0" alt="cross stitch with a bowl of salad and the words 'You don't win friends with salad' stitched underneath" src="./images/blog_img/cross_stitch.png" width="100%" height="100%">
```

Comment:
You're doing inline styles here with the border and sizing. Do these with CSS instead. :)